api/resources_portal/test/views/test_material_request.py

```diff
@@ -103,11 +103,22 @@ def test_post_request_from_unauthenticated_forbidden(self):
         response = self.client.post(self.url, self.material_request_data, format="json")
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
-    def test_get_request_succeeds(self):
+    def test_get_request_from_sharer_succeeds(self):
         self.client.force_authenticate(user=self.sharer)
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_get_request_from_requester_succeeds(self):
+        self.client.force_authenticate(user=self.request.requester)
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_get_request_filters(self):
```

Comment:
Another test where the filter returns one or more requests would be good. You could even just add `self.assertEqual(response.json()["count"], 1)` at the end of the previous two tests if you wanted.